transact/src/main/java/dev/dbos/transact/execution/ExecuteWorkflowOptions.java

```diff
@@ -0,0 +1,56 @@
+package dev.dbos.transact.execution;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Objects;
+
+/** Options for executing a workflow, including identifier, queue, and timeout settings, */
+public record ExecuteWorkflowOptions(
```

Comment:
This seems redundant with StartWorkflowOptions. Why do we need both? Are event listeners allowed to set workflow deadline? Why?